libs/utils/src/votes.ts

```diff
@@ -136,104 +134,90 @@ export function getVotingMethodForCastVoteRecord(
 interface TallyParams {
   election: Election;
   votes: VotesDict[];
-  filterContestsByParty?: PartyId;
 }
 export function tallyVotesByContest({
   election,
   votes,
-  filterContestsByParty,
```

Comment:
Yeah I think things are confusing as before filtering by party really was a filter on contests and not cvrs (it happens to be that those contests are only for some cvrs), and now since you are including nonpartisan races on the tally reports filtering by party is filtering by cvrs.